pom.xml

```diff
@@ -2,18 +2,18 @@
 		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
 	<modelVersion>4.0.0</modelVersion>
-	<groupId>net.hifis.cloud.marketplace</groupId>
-	<artifactId>hifis-marketplace</artifactId>
+	<groupId>de.helmholtz.marketplace</groupId>
+	<artifactId>helmholtz-marketplace-webapp</artifactId>
 	<version>0.0.1-SNAPSHOT</version>
 	<packaging>pom</packaging>
 	<inceptionYear>2020</inceptionYear>
 
-	<name>HIFIS Marketplace</name>
-	<url>https://www.hifis.net</url>
```

Comment:
I am somewhat unsure about this URL. I tend to stay with the HIFIS URL for the time of the development, since you will not get any detailed information about the development of the platform on the general Helmholtz website.